frontend/src/components/associations/marketplace/ProductsAdministration.tsx

```diff
@@ -13,32 +14,70 @@ export const AssociationMarketplaceProductAdministration = ({
-            <Row>
-                <Pagination
-                    apiKey={["products.list", marketplaceId, { page_size: 8 }]}
-                    apiMethod={api.products.list}
-                    render={(products, controlbar) => (
-                        <>
-                            <Row>
-                                {products.map((product) => (
-                                    <Product
-                                        product={product}
-                                        key={product.id}
-                                        additionalContent={
-                                            <Link
-                                                to={`/associations/${marketplaceId}/magasin/produits/${product.id}/modifier`}
-                                            >
-                                                Modifier
-                                            </Link>
-                                        }
-                                    />
-                                ))}
-                            </Row>
-                            {controlbar}
-                        </>
-                    )}
-                />
-            </Row>
+
+            <Pagination
+                apiKey={["products.list", marketplaceId, { page_size: 8 }]}
+                apiMethod={api.products.list}
+                paginationControlProps={{
+                    className: "justify-content-center mt-5",
+                }}
+                render={(products, paginationControls) => (
+                    <>
+                        <Card>
+                            <Table
+                                columns={columns(marketplaceId)}
+                                data={products}
+                            />
+                        </Card>
+                        {paginationControls}
+                    </>
+                )}
+            />
         </>
     );
 };
+
+const columns = (marketplaceId) => [
+    {
+        key: "product",
+        header: "Product",
+        render: (product) => product.name,
+    },
+    {
+        key: "price",
+        header: "Prix",
+        cellClassName: "text-muted",
+        render: (product) => product.price + "€",
+    },
+    {
+        key: "description",
+        header: "Description",
+        cellClassName: "flex-shrink-1",
+    },
+    {
+        key: "stocks",
+        header: "Stocks",
+        cellClassName: "text-muted",
+        render: (product) =>
+            product.numberLeft > -1
+                ? ` ${product.numberLeft} ${decidePlural(
+                      product.numberLeft,
+                      "restant",
+                      "restants"
+                  )}`
+                : "Stocks infinis",
+    },
+    {
+        key: "action",
+        header: "Actions",
+        render: (product) => (
+            <Link
+                to={`/associations/${marketplaceId}/magasin/produits/${product.id}/modifier`}
```

Comment:
Would it be possible to have an icon button here?
I find the default Link style a bit sad. 😁